src/Addons/User/AddonsUser.php

```diff
@@ -109,9 +85,7 @@ public function getEmail(): array
             if ($this->isConnectedOnPsAccounts()) {
                 $email = $this->accountsDataProvider->getAccountsUserEmail();
             } elseif ($this->hasAccountsTokenInSession()) { // Connected on ps_accounts with session
```

Comment:
Do e need the elseif ?

Reply:
Yes, we need that check on session because the accounts token is set on session when executing actions on modules (API request)